### ISSUE TITLE
[0.8.8] Influxdb service crashed and disappeared from process list

### ISSUE THREAD

**Issue by shaileshswapnil** [influxdb/influxdb#3946]
Hi All,
I was running continuous I/O in my application and was recording all the points which are being generated. Suddenly after a long time, influxDB service got stopped working. Suddenly it got disappeared from the ps command. Now If I am trying to start the service, it is starting fine but after few seconds, it is failing again. No idea about this behavior of influxDB.

I am using influxDB with grafana for visualization.

Is it any type of memory issue or open file limit in unix related problem?

OS: CentOS 6.6
InfluxDB: 0.8.8-1

**Comment by jwilder** [comment]
I would suggest checking the server log and you your syslog ('/var/log/syslog') for some indication as to why it crashed.

**Comment by beckettsean** [comment]
@shaileshswapnil can you share the InfluxDB logs or have you had a chance to check syslog?

**Comment by shaileshswapnil** [comment]
yes I checked syslog, it had no info. So I investigated more, still couldn't find any reason for this type of behavior. But one possible reason could be, as we have distributed system with hundreds of daemons running all the time, and checking influxDB and Grafana to manage its working, that might killed have this influxDB process somehow. But still this problem is not traced, and it is happening often, with no syslog or crash report.

So it might not be the problem of influxDB, I guess.

**Comment by shaileshswapnil** [comment]
yes I checked syslog, it had no info. So I investigated more, still couldn't find any reason for this type of behavior. But one possible reason could be, as we have distributed system with hundreds of daemons running all the time, and checking influxDB and Grafana to manage its working, that might have killed this influxDB process somehow. 

But still this problem is not traced, and it is happening often, with no syslog or crash report.
So this might not be the issue of influxDB, I guess.
